tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.cs

```diff
@@ -165,5 +167,84 @@ public void AddResponseHeadersToSpanAndCleanup()
         _httpTransport.DisposeAdditiveContext();
     }
 
+    internal static Dictionary<string, object> ExtractCookiesFromRequest(HttpRequest request)
+    {
+        var cookies = RequestDataHelper.GetCookies(request);
+        var cookiesDic = new Dictionary<string, object>();
+
+        if (cookies != null)
+        {
+            for (var i = 0; i < cookies.Count; i++)
+            {
+#if NETCOREAPP || NETSTANDARD
```

Comment:
maybe we could use both `SecurityCoordinator.Core` and `SecurityCoordinator.Framework` to avoid these ifs?